universal-login-react/src/ui/commons/WaitingForTransaction.tsx

```diff
@@ -8,9 +8,10 @@ import {getStyleForTopLevelComponent} from '../../core/utils/getStyleForTopLevel
 export interface WaitingForTransactionProps extends WaitingForProps {
   relayerConfig: PublicRelayerConfig;
   transactionHash?: string;
+  info?: string;
```

Comment:
Debatable idea - Maybe make it required? If nothing is to be displayed, then just pass empty string `''`